interfaces/builtin/iio_test.go

```diff
@@ -36,8 +36,10 @@ type IioInterfaceSuite struct {
 	iface interfaces.Interface
 
 	// OS Snap
-	testSlot1     *interfaces.ConnectedSlot
-	testSlot1Info *snap.SlotInfo
+	testSlot1           *interfaces.ConnectedSlot
+	testSlot1Info       *snap.SlotInfo
+	testSlotCleaned     *interfaces.ConnectedSlot
```

Comment:
Why is this called testSlotCleaned if it is looking at `test-port-unclean`.
Perhaps there's a good reason but when there are lots of plugs and slots declared being obvious might be preferential.